.github/workflows/dotcom-acceptance-tests-manual.yml

```diff
@@ -0,0 +1,109 @@
+name: Dotcom Acceptance Tests (manual)
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+
+  acceptance-tests-anonymous:
+    runs-on: ubuntu-latest
+    if: contains(join(github.event.pull_request.labels.*.name, ', '), 'test')
+    steps:
+      - name: Parse Args
+        id: args
+        run: |
+          echo "::set-output name=run_allowed::$(
+            jq -rc .label.name $GITHUB_EVENT_PATH | cut -d/ -f 2
+          )"
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          fetch-depth: 2
```

Comment:
Out of curiosity, what's the reason for `fetch-depth: 2` here?

Reply:
Will revisit here with more context, seems architectural though.